jme3-terrain/src/main/java/com/jme3/terrain/geomipmap/TerrainQuad.java

```diff
@@ -1306,7 +1306,7 @@ private int findQuadrant(int x, int y) {
     public void setLocked(boolean locked) {
         for (int i = 0; i < this.getQuantity(); i++) {
             if (this.getChild(i) instanceof TerrainQuad) {
-                ((TerrainQuad) getChild(i)).setLocked(locked);
+                ((Terrain) getChild(i)).setLocked(locked);

```

Comment:
This looks wrong/useless to me:
Even when setLocked is implemented on Terrain instead of TerrainQuad, this cast will confuse people.
Specifically one should look if the instanceof Check can be extended to Terrain then.

Reply:
Yeah, idiomatically, you should cast to the class you checked for.

Reply:
This delta wasn't supposed to be included in the PR. My mistake. I'll take it out.